src/reducers/related-view.test.ts

```diff
@@ -0,0 +1,35 @@
+import {RELATED_VIEWS_HIDE_TOGGLE} from '../actions/related-views';
+import {RelatedViews} from '../models/related-views';
+import {relatedViewsReducer} from './related-views';
+
+describe('reducers/related-views', () => {
+  it('should toggle relatedViewsToggler to hide related-views', () => {
+    const expectedRelatedViews: RelatedViews = {
+      isHidden: true
+    };
+    expect(relatedViewsReducer(
+      {
+        isHidden: false
+      },
+      {
+        type: RELATED_VIEWS_HIDE_TOGGLE
+      }
+    )).toEqual(expectedRelatedViews);
+  });
+});
+
+describe('reducers/related-views', () => {
```

Comment:
Usually, we have just one 'describe' for multiple test cases that are under the same hierarchy